code/modules/ruins/lavalandruin_code/elephantgraveyard.dm

```diff
@@ -0,0 +1,240 @@
+	impressiveness = 18 // Carved from the bones of a massive creature, it's going to be a specticle to say the least
+	layer = ABOVE_ALL_MOB_LAYER
+
+/obj/structure/statue/bone/rib
+	name = "collosal rib"
+	desc = "It's staggering to think that something this big could have lived, let alone died."
+	oreAmount = 4
+	icon = 'icons/obj/statuelarge.dmi'
+	icon_state = "rib"
+
+/obj/structure/statue/bone/skull
+	name = "collosal skull"
+	desc = "The gaping maw of a dead, titanic monster."
+	oreAmount = 12
+	icon = 'icons/obj/statuelarge.dmi'
+	icon_state = "skull"
+
+/obj/structure/statue/bone/skull/half
+	desc = "The gaping maw of a dead, titanic monster. This one is cracked in half."
+	oreAmount = 6
+	icon = 'icons/obj/statuelarge.dmi'
+	icon_state = "skull-half"
+
+//***Wasteland and cracked earth floor tiles, subtype of strong stone to use here.
+/turf/open/floor/plating/asteroid/basalt/wasteland //Like a more fun version of living in Arizona.
+	name = "cracked earth"
+	icon = 'icons/turf/floors.dmi'
+	icon_state = "wasteland"
+	environment_type = "wasteland"
+	baseturfs = /turf/open/floor/plating/asteroid/basalt/wasteland
+	digResult = /obj/item/stack/ore/glass/basalt
+	initial_gas_mix = LAVALAND_DEFAULT_ATMOS
+	slowdown = 0.5
+	floor_variance = 30
+
+/turf/closed/mineral/strong/wasteland
+	name = "Ancient dry rock"
+	color = "#B5651D"
+	environment_type = "wasteland"
+	turf_type = /turf/open/floor/plating/asteroid/basalt/wasteland
+	baseturfs = /turf/open/floor/plating/asteroid/basalt/wasteland
+	initial_gas_mix = LAVALAND_DEFAULT_ATMOS
+	defer_change = 1
+	smooth_icon = 'icons/turf/walls/rock_wall.dmi'
+
+/turf/closed/mineral/strong/wasteland/gets_drilled(user)
+	if(prob(10))
+		new /obj/item/stack/ore/iron(src, 1)
+		new /obj/item/stack/ore/glass(src, 1)
+		new /obj/effect/decal/remains/human(src, 1)
+	else
+		new /obj/item/stack/sheet/bone(src, 1)
+	var/flags = NONE
+	if(defer_change)
+		flags = CHANGETURF_DEFER_CHANGE
+	ScrapeAway(flags=flags)
+	addtimer(CALLBACK(src, .proc/AfterChange), 1, TIMER_UNIQUE)
+	playsound(src, 'sound/effects/break_stone.ogg', 50, TRUE)
+
+/turf/open/floor/plating/asteroid/basalt/wasteland/Initialize()
+	.=..()
+	if(prob(floor_variance))
+		icon_state = "[environment_type][rand(0,6)]"
+
+//***Oil well puddles.
+/obj/structure/sink/oil_well	//You're not going to enjoy bathing in this...
+	name = "oil well"
+	desc = "A bubbling pool of oil.This would probably be valuable, had bluespace technology not destroyed the need for fossil fuels 200 years ago."
+	icon = 'icons/obj/watercloset.dmi'
+	icon_state = "puddle-oil"
+	dispensedreagent = /datum/reagent/fuel/oil
+
+/obj/structure/sink/oil_well/attack_hand(mob/M)
+	icon_state = "puddle-oil-splash"
+	. = ..()
+	icon_state = "puddle-oil"
+
+/obj/structure/sink/oil_well/attackby(obj/item/O, mob/user, params)
+	icon_state = "puddle-oil-splash"
+	. = ..()
+	icon_state = "puddle-oil"
+
+/obj/structure/sink/oil_well/deconstruct(disassembled = TRUE)
```

Comment:
Again the default without the signal ? Same thing as with the mineral subtype. Wrap dropping the materials in a proc and override that instead.

Reply:
This is still an issue.